jest.setup.js

```diff
@@ -11,14 +11,14 @@ jest.mock('@storybook/addon-info', () => ({
   setDefaults: () => {},
 }));
 
-// Next Router
-jest.mock('next/router', () => ({
-  push: () => {},
-  prefetch: () => {},
-  replace: () => {},
-  route: '/mock-route',
-  withRouter: Component => Component,
-}));
+// Next Router (mocking this isn't inherently useful)
```

Comment:
It is for a few specific stories relying on link buttons (like the reusable "Join Us" section)

Reply:
k